cluster/addons/fluentd-elasticsearch/es-pod.yml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1beta1
+kind: Pod
```

Comment:
@thockin: I am working on replication controllers. I will add them along with a few other enhancements to monitoring.